### PR TITLE
Add DRL-1.1

### PR DESCRIPTION
**Pull request by nasbench**
Add new update to the DRL license as per discussion in #1992

### CODE REVIEW

File: src/DRL-1.1.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+<license licenseId="DRL-1.1" name="Detection Rule License 1.1" isOsiApproved="false" listVersionAdded="3.12">
```
Comment (xsuchy) [inline src/DRL-1.1.xml:3]: VersionAdded should be 3.23

Comment (nasbench) [inline src/DRL-1.1.xml:3]:
```suggestion
<license licenseId="DRL-1.1" name="Detection Rule License 1.1" isOsiApproved="false" listVersionAdded="3.23">
```

Comment (nasbench) [reply src/DRL-1.1.xml]: Modified

**Comment by jlovejoy** [comment]
Thanks @nasbench for the PR and thanks @xsuchy  for the review!

**Review by xsuchy — APPROVED** [review]
LGTM me - from technical POV

**Pull request merged**